eos-tech-support/eos-fix-ostree-repo

```diff
@@ -249,7 +249,7 @@ def mark_commits_partial(repo):
         if objtype != OSTree.ObjectType.COMMIT:
             continue
         _, commit, state = repo.load_commit(checksum)
-        if state == OSTree.RepoCommitState.REPO_COMMIT_STATE_PARTIAL:
+        if state == OSTree.RepoCommitState.PARTIAL:
```

Comment:
I think we'll need to add compatibility for the old names since we distribute this script to people on old Endless installs. See all the other junk earlier in the script trying to cope with missing symbols.

Reply:
⇒ #505